### ISSUE TITLE
Angular 14 update

### ISSUE THREAD

**Issue by yonatancohen** [danrevah/ngx-pipes#226]
I'll Really appreciate if you have the time to upgrade to latest angular.

**Comment by bitsmuggler** [comment]
@yonatancohen As a **temporary** workaround:  you can make it run under Angular 14.0 by adding the ``overrides`` for ngx-pipes section to package.json

```
  "overrides": {
    "ngx-pipes": {
      "@angular/core": "^14.0.0"
    }
  }

```

This worked for me.

**Comment by danrevah** [comment]
Released as part of ngx-pipes v3.1.0
